torch/distributed/_tensor/ops/_math_ops.py

```diff
@@ -317,6 +317,7 @@ def common_reduction_strategy(
     aten.max.default: "max",
     aten.max.dim: "max",
     aten.max.out: "max",
+    aten.isinf.default: "max",
```

Comment:
hmmm I just checked the doc https://pytorch.org/docs/stable/generated/torch.isinf.html#torch.isinf 
this operator seems to be a pointwise operator, so I think it should be added to https://github.com/pytorch/pytorch/blob/main/torch/distributed/_tensor/ops/_pointwise_ops.py#L59

So it would not produce Parital(max) unless later there's a reduction operator happened after calling isinf